code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -618,15 +618,15 @@ var/robot_arm = /obj/item/robot_parts/l_arm
 			if(!user.unEquip(W))
 				return
 			build_step++
-			to_chat(user, "<span class='notice'>You add the bikehorn to [src]! Honk!</span>")
+			to_chat(user, "<span class='notice'>You add the bikehorn to [src] Honk!</span>")
```

Comment:
(Why is the exclamation point here removed, if you removed the period above?)

Reply:
Fixed.